### ISSUE TITLE
[Feature Request] Add support for qbittorrent enhanced edition

### ISSUE THREAD

**Issue by clsty** [VueTorrent/VueTorrent#867]
E.g. [Qbittorrent enhanced edition](https://github.com/c0re100/qBittorrent-Enhanced-Edition) has a "Automatically update public trackers list" under Settings -> BitTorrent, and vuetorrent does not show that (default WebUI does).

**Comment by Larsluph** [comment]
Do you know if there's a way to determinate which version (or fork) of qbit is installed through the API ?

**Comment by clsty** [comment]
> Do you know if there's a way to determinate which version (or fork) of qbit is installed through the API ?

Sorry, I don't know.

@c0re100 Hi, could you please give some hints or help on this topic? Thanks in advance!

**Comment by c0re100** [comment]
> Do you know if there's a way to determinate which version (or fork) of qbit is installed through the API ?

API `/api/v2/app/version` should return qbittorrent(enhanced) version number
e.g.: `v4.5.4`(official) or `v4.5.4.10`(enhanced)

**Comment by killerprojecte** [comment]
😎This should be brought up again

**Comment by Larsluph** [comment]
I tried installing the enhanced version. I haven't seen any way to differentiateit from the official version. Without it I doubt that this is feasible.

**Comment by killerprojecte** [comment]
> I tried installing the enhanced version. I haven't seen any way to differentiateit from the official version. Without it I doubt that this is feasible.

You can determine whether it is an enhanced version by setting the fields returned by the interface

**Comment by killerprojecte** [comment]
@Larsluph [Modified Commit](https://github.com/killerprojecte/VueTorrent-Enhanced/commit/f0521bbd450e93557c754551e06fec2cbcbe2f95#diff-fcce84c6804c7878cb13cd723040cebddf69219e9c9df2732c1ba59d95309abd)

I tried to implement the related functions in my fork, but I did not distinguish between the two versions. 
The difference here is that there are two more JSON keys. 
You only need to determine whether the JSON object returned by the API contains a specific key (two or any one) to determine whether it is an enhanced version.
